### PR TITLE
ENH:  Add deep flash.

### PR DESCRIPTION
**Pull request by ntustison**

### CODE REVIEW

**Comment by ntustison** [comment]
Hey @stnava,

So I'm trying to test this and I'm getting error below [when setting up for the first time](https://github.com/stnava/ANTsPyT1w#first-time-setup).  I thought it might have been the changes I made but I get the same error when I installed ANTsPyT1w for the first time on another machine.  Let me know if you have any ideas.  Thanks.

```
>>> import antspyt1w
>>> antspyt1w.get_data(force_download=True)
Downloading data from https://ndownloader.figshare.com/articles/14766102/versions/23
263929856/263923770 [==============================] - 98s 0us/step
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ntustison/Pkg/ANTsPyT1w/antspyt1w/get_data.py", line 115, in get_data
    raise ValueError('File doesnt exist. Options: ' , os.listdir(DATA_PATH))
ValueError: ('File doesnt exist. Options: ', ['28497-00000000-T1w-04.nii.gz', 'CIT168_basal_forebrain_adni_prob_1_right.nii.gz', 'dlbs.csv', 'T_template0_LobesBstem.csv', 'CIT168_basal_forebrain_prob_2_right.nii.gz', 'T_template0.nii.gz', 'CIT168_basal_forebrain_adni_prob_1_left.nii.gz', 'T_template0_Symmetric.nii.gz', '28575-00000000-T1w-07.nii.gz', 'wmh_evidence.csv', '28478-00000000-T1w-03.nii.gz', 'T_template0_LobesBstem.nii.gz', 'CIT168_T1w_700um_pad.nii.gz', 'det_atlas_25_pad_LR_adni.nii.gz', 'CIT168_Reinf_Learn_v1_label_descriptions_pad.csv', 'CIT168_MT_Slab.nii.gz', 'CIT168_basal_forebrain_adni_prob_2_left.nii.gz', 'run_dlbs.py', 'CIT168_MT_Slab_adni.nii.gz', 'lobes.csv', 'tissues.csv', 'ADNI-073_S_4300-20140107-T1w-000.nii.gz', 'dkt.csv', 'README', 'T_template0_WMP.nii.gz', 'CIT168_basal_forebrain_prob_2_left.nii.gz', 'det_atlas_25_pad_LR.nii.gz', 'hemisphere.csv', 'CIT168_T1w_700um_pad_adni_brainstem.csv', 'PPMI-3803-20120814-MRI_T1-I340756.nii.gz', '28386-00000000-T1w-01.nii.gz', 'T_template0_LR.nii.gz', 'T_template0_SymmetricLR.nii.gz', 'wm_major_tracts.nii.gz', 'CIT168_T1w_700um_pad_adni.nii.gz', 'CIT168_basal_forebrain_prob_1_right.nii.gz', '28542-00000000-T1w-06.nii.gz', 'CIT168_basal_forebrain_adni_prob_2_right.nii.gz', 'wm_major_tracts.csv', '28523-00000000-T1w-05.nii.gz', 'simwmhseg.h5', '28405-00000000-T1w-02.nii.gz', 'simwmdisc.h5', 'CIT168_basal_forebrain_prob_1_left.nii.gz', 'CIT168_T1w_700um_pad_adni_brainstem.nii.gz', '28364-00000000-T1w-00.nii.gz'])
```

**Comment by stnava** [comment]
I don't think this is an error.  it's just downloading the data and saying you requiested a file ( ie None ) that does not exist.

**Comment by ntustison** [comment]
Okay, awesome.  I ran into something similar earlier midway during the test run but I hadn't forced a reload of the data.  I mistakingly assumed they were related.  Let me try again and see if it runs now.

**Comment by ntustison** [comment]
Okay, @stnava, see below.  It looks I was able to integrate ``deep_flash`` with [``do_preprocessing = False``](https://github.com/stnava/ANTsPyT1w/blob/AddDeepFlash/antspyt1w/get_data.py#L517) and the example in the readme runs as it should, I believe.   So now I just need some direction as to what additional finishing touches you would like to see.  [Here's the main function.](https://github.com/stnava/ANTsPyT1w/blob/AddDeepFlash/antspyt1w/get_data.py#L477-L544)  I called it ``deep_mtl`` but let me know if you want to change it.  Also, note that I'm [returning](https://github.com/stnava/ANTsPyT1w/blob/AddDeepFlash/antspyt1w/get_data.py#L540-L543) only the segmentation image and the probability images for the MTL parcellation.  I'm not returning the other outputs as I figure that would be for more advanced usage.  I'm guessing you might want to also include output from ``ants.label_geometry_measures`` but I'll wait for your input.

Screen Shot 2021-11-05 at 5 01 02 PM

**Comment by stnava** [comment]
we need one more thing a csv file of the form :

```
Label,Description
0,background
4,left lateral ventricle
...
```

so we can have interpretable outputs from this.

**Comment by ntustison** [comment]
Okay @stnava, I think this [latest commit](https://github.com/stnava/ANTsPyT1w/pull/4/files#diff-99b28a95190e1862c442f1ee58c647ebd12628d7e63c12113e3f622d86be0a90R556-R563) adds that.  Let me know if I need something else.

**Comment by ntustison** [comment]
oops, @stnava, I just noticed that you only need the label and description (not everything else).  Let me fix that.

**Comment by stnava** [comment]
looks good
